### ISSUE TITLE
space separated, all lowercase is in reality comma seperated

### ISSUE THREAD

**Issue by deralexxx** [viper-framework/viper#730]
https://github.com/viper-framework/viper/blob/5cf700761d904d9002a272164735c2ed2b4e0df9/viper/web/viperapi/serializers.py#L289

@frennkie is that a wrong helping text? I played with the API today and:

`data = {
        "file_name": filename,
        "tag_list":"tag1 tag2,tag3"

    }`

I tried both comma separated and whitespace and only comma separated are respected as different tags.

**Comment by frennkie** [comment]
I just tired to think how this happend.. I believe I wanted to allow tags with spaces and for that reason switched from space separated to comma separated (and forgot to change the help description).

```
    # tag_list expects a comma separated list of tags (tags are always all lowercase)
    def validate_tag_list(self, value):
        return [x.lower() for x in value.strip().split(",")]
```

**Comment by frennkie** [comment]
So, the way how the CLI works prevents tags with spaces... I just tried that out.

Anyway I would suggest to keep the API field with comma separated. To enforce the "no-spaces" behavior also on the API (and Web) I could add another validation steps... what do you think?

**Comment by frennkie** [comment]
@deralexxx fix with 911298decd4d96668f294cc1ab1cdbc3299d93dc please check.

**Comment by deralexxx** [comment]
cool thank you for the fast response, yes that works
